visualc-gdk/FNA3D.vcxproj

```diff
@@ -56,11 +56,6 @@
     <ClInclude Include="..\src\FNA3D_Driver.h" />
     <ClInclude Include="..\src\FNA3D_Tracing.h" />
   </ItemGroup>
-  <ItemGroup>
```

Comment:
Should these references be removed as part of the move to SDL3?

Reply:
Yeah, we need some way of linking with both SDL2/3 (for now) so I'm not sure what this should look like. Though I don't remember making this specific change, the project reference might have been removed automatically by my IDE.